subsys/net/l2/wifi/wifi_utils.c

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Utility functions to be used by the Wi-Fi subsytem.
+ */
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(net_wifi_utils, LOG_LEVEL_INF);
```

Comment:
We shouldn't set a fixed log level, the module should either specify its own log level in Kconfig, or reuse one of the existing (would this fall under CONFIG_NET_L2_WIFI_MGMT_LOG_LEVEL umbrella?).

Reply:
Thanks. Yeah it would make sense to have it under the CONFIG_NET_L2_WIFI_MGMT_LOG_LEVEL umbrella. Updated now.